code/__DEFINES/subsystems.dm

```diff
@@ -43,9 +43,10 @@
 // Subsystems shutdown in the reverse of the order they initialize in
 // The numbers just define the ordering, they are meaningless otherwise.
 
-#define INIT_ORDER_DBCORE 18
-#define INIT_ORDER_BLACKBOX 17
-#define INIT_ORDER_SERVER_MAINT 16
+#define INIT_ORDER_DBCORE 19
+#define INIT_ORDER_BLACKBOX 18
+#define INIT_ORDER_SERVER_MAINT 17
+#define INIT_ORDER_RNG 16
```

Comment:
I'm going to need to do an MC change to make PreInit() respect init_order because currently it doesn't